README.md

```diff
@@ -89,3 +89,26 @@ Of these variables,
 * $btn-secondary-hover-border-color
 
 are required. All the other variables are optional.
+
+To use on the devstack, you need to create `lms/static/sass/common-variables.scss`
```

Comment:
Nit: inconsistent styling - other lines of this doc don't use 80-character lines.

Reply:
I've increased the line limit to 120, but I generally feel there should be a limit.